engines/sci/engine/script_patches.cpp

```diff
@@ -112,6 +112,7 @@ static const char *const selectorNameTable[] = {
 	"startAudio",   // King's Quest 6 CD / Laura Bow 2 CD for audio+text support
 	"modNum",       // King's Quest 6 CD / Laura Bow 2 CD for audio+text support
 	"has",          // King's Quest 6, GK1
+	"modeless",     // King's Quest 6
```

Comment:
Please change the description to King’s Quest 6 CD, like above

Reply:
Good catch, comment updated.